src/Admin/Vue.php

```diff
@@ -385,30 +389,22 @@ public function payplug_section_status() {
 					"enable_debug_description" => __("payplug_section_status_debug_description", "payplug"),
 				]
 			],
-			"options" => [
-				"type" => "-warning",
-				"name" => "requirements",
-				"options" => [
-					[
-						"status" => $payplug_requirements->valid_curl(),
-						"text" => __("payplug_section_status_curl", "payplug")
-					],
-					[
-						"status" => $payplug_requirements->valid_php(),
-						"text" => __("payplug_section_status_php", "payplug")
-					],
-					[
-						"status" => $payplug_requirements->valid_openssl(),
-						"text" => __("payplug_section_status_ssl", "payplug")
-					],
-					[
-						"status" => $payplug_requirements->valid_currency(),
-						"text" => __("payplug_section_status_currency", "payplug")
-					],
-					[
-						"status" => $payplug_requirements->valid_account(),
-						"text" => __("payplug_section_status_account", "payplug")
-					]
+			"requirements" => [
```

Comment:
missing more checks

`$this->valid_php()
&& $this->valid_curl()
&& $this->valid_openssl()
&& $this->valid_currency()
&& $this->valid_account();`